src/account/__tests__/accountReducers-test.js

```diff
@@ -13,10 +15,13 @@ describe('accountReducers', () => {
   describe('REALM_ADD', () => {
     test('if no account with this realm exists, insert new account in front', () => {
       const prevState = [{ realm: '1' }, { realm: '2' }];
```

Comment:
done 👍 (looked at https://github.com/substack/deep-freeze/blob/master/index.js#L13 to check)